examples/textConnect/desktops/desktop.js

```diff
@@ -96,39 +99,27 @@
         return selectionArry;
     }
 
-
     function connectField(record) {
-
-        // 各結合項目の処理
-        // Every iteration, one resolve field is calculated based on its delimiter and selection fields.
+        // Every iteration, one resolve field is calculated based on it's delimiter and selection fields.
         for (var i = 1; i < 4; i++) {
             var cdcopyfield = CONF['copyfield' + i];
             var cdbetween = CONF['between' + i];
             var selectionArry = createSelectionArry();
-            var rawTextArray = fieldValues(record, selectionArry[i - 1]); // array of text field values
-
-            // Filter rawTextArray to only include non empty strings
-            var filteredTextArray = rawTextArray.filter(function(text) {
-                return text !== "";
-            });
-
-            // Special cases for delimiters of '&nbsp;' and '&emsp;'
+            var joinText = fieldValues(record, selectionArry[i - 1]);
```

Comment:
Please do not remove logic that was implemented in #135 .
This is the logic for skipping un-needed delimiters when a value for a Field to Connect is empty.

Reply:
Reference
https://github.com/kintone/plugin-examples/blob/b4b3dc28c502f3d398397d2958b50e1d0af1dd9f/examples/textConnect/desktops/desktop.js#L100-L127

Reply:
Revert the changes back.